diagrams/ibm/security.py

```diff
@@ -0,0 +1,11 @@
+# This module is automatically generated by autogen.sh. DO NOT EDIT.
```

Comment:
Aren't there any security nodes :) ?

Reply:
@mingrammer Please review. I added more diagrams.

Reply:
@mingrammer can you approve this week end ?